src/main/webapp/app/view/FreetextAnswerPanel.js

```diff
@@ -124,7 +124,32 @@ Ext.define('ARSnova.view.FreetextAnswerPanel', {
 			docked: 'top',
 			ui: 'light',
 			title: Ext.util.Format.htmlEncode(this.questionObj.subject),
-			items: [this.backButton]
+			items: [this.backButton, {
```

Comment:
It's best to define a var for the export button (like the backButton).

Besides, i'd suggest relocating the code for the export into the QuestionExport Controller for a cleaner view-code.